### ISSUE TITLE
Not using shell set in /etc/passwd

### ISSUE THREAD

**Issue by Zlopez** [debarshiray/fedora-toolbox#18]
fedora-toolbox always start with `/bin/bash` instead of shell, that is set for the user in `/etc/passwd`

In my case, I have this in `/etc/passwd`
```
zlopez:x:1000:1000::/home/zlopez:/usr/bin/fish
```
And I have `fish` installed in fedora-toolbox, but it always starts with `/bin/bash` instead.

I noticed that even the `tmux` inside is starting `/bin/bash` instead of `/usr/bin/fish`.

**Comment by debarshiray** [comment]
Which version of `fedora-toolbox` do you have? This should be fixed in `0.0.2`.

There are updates on their way to the stable Fedora repositories:
* https://bodhi.fedoraproject.org/updates/FEDORA-2018-0c00b549ec
* https://bodhi.fedoraproject.org/updates/FEDORA-2018-e8d4a69e8b

**Comment by Zlopez** [comment]
I cloned `master`.
I did `git pull` before I used `./fedora-toolbox enter` to test this.

**Comment by Zlopez** [comment]
I tried to recreate the fedora-toolbox image, because `podman ps` shows that the exec command was `/bin/sh`.
So I stopped it using `podman stop` and than removed it using `podman rm`.
Now when I try `./fedora-toolbox -v create` I'm getting this:
```
error looking up container "fedora-toolbox-zlopez:29": no container with name or ID fedora-toolbox-zlopez:29 found: no such container
invalid host path, must be an absolute path ""
./fedora-toolbox: failed to create container fedora-toolbox-zlopez:29
```

**Comment by mildred** [comment]
Version 0.0.2 (packaged in Fedora 29) is keeping the shell I configured in /etc/passwd.

**Comment by debarshiray** [comment]
> Now when I try ./fedora-toolbox -v create I'm getting this:

@Zlopez It was likely caused by the `podman` and `buildah` turbulence that had crept into Fedora for some weeks. It should be fixed now, but either way isn't related to `fedora-toolbox` itself.

As @mildred confirmed, I do expect this issue to be fixed in `0.0.2`. Closing.
